Formula/shush.rb

```diff
@@ -0,0 +1,17 @@
+class Shush < Formula
+  desc "Encrypt and decrypt secrets using the AWS Key Management Service"
+  homepage "https://github.com/realestate-com-au/shush"
+  url "https://github.com/realestate-com-au/shush/archive/refs/tags/v1.5.3.tar.gz"
+  sha256 "07eed7f6fa34b0cadf64e5dfde752f12fa038293765eef35d43790c479e72fc6"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin.to_s + "/shush"
+  end
+
+  test do
+    system "#{bin}/shush", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.